spec/stored_queries/taxonomy/taxonomy_ncbi_species_no_sort.yaml

```diff
@@ -0,0 +1,50 @@
+# Search ncbi_taxon collection for species/strains by scientific name
```

Comment:
New ncbi species/strain search, but with no sorting, for when the length of search text is low